File: snippets/scala/src/main/scala/Analytics.scala
package algoliasearch.methods.snippets

import scala.concurrent.duration.Duration

import algoliasearch.api.AnalyticsClient
import algoliasearch.analytics.*

import org.json4s.*
import org.json4s.native.JsonParser.*
import scala.concurrent.{Await, ExecutionContextExecutor}

class SnippetAnalyticsClient {
  implicit val ec: ExecutionContextExecutor = scala.concurrent.ExecutionContext.global
  implicit val formats: Formats = org.json4s.DefaultFormats

  /** Snippet for the customDelete method.
    *
    * allow del method for a custom path with minimal parameters
    */
  def snippetForAnalyticsClientCustomDelete(): Unit = {
    // >SEPARATOR customDelete
    // Initialize the client
    val client = AnalyticsClient(appId = "YOUR_APP_ID", apiKey = "YOUR_API_KEY", region = Option("YOUR_APP_ID_REGION"))

    // Call the API
    val res = client.customDelete[JObject](
      path = "/test/minimal"
    )

    // Use the response
    val value = Await.result(res, Duration(100, "sec"))
    // SEPARATOR<
  }

  /** Snippet for the customGet method.
    *
    * allow get method for a custom path with minimal parameters
    */
  def snippetForAnalyticsClientCustomGet(): Unit = {
    // >SEPARATOR customGet
    // Initialize the client
    val client = AnalyticsClient(appId = "YOUR_APP_ID", apiKey = "YOUR_API_KEY", region = Option("YOUR_APP_ID_REGION"))

    // Call the API
    val res = client.customGet[JObject](
      path = "/test/minimal"
    )

    // Use the response
    val value = Await.result(res, Duration(100, "sec"))
    // SEPARATOR<
  }

  /** Snippet for the customPost method.
    *
    * allow post method for a custom path with minimal parameters
    */
  def snippetForAnalyticsClientCustomPost(): Unit = {
    // >SEPARATOR customPost
    // Initialize the client
    val client = AnalyticsClient(appId = "YOUR_APP_ID", apiKey = "YOUR_API_KEY", region = Option("YOUR_APP_ID_REGION"))

    // Call the API
    val res = client.customPost[JObject](
      path = "/test/minimal"
    )

    // Use the response
    val value = Await.result(res, Duration(100, "sec"))
    // SEPARATOR<
  }

  /** Snippet for the customPut method.
    *
    * allow put method for a custom path with minimal parameters
    */
  def snippetForAnalyticsClientCustomPut(): Unit = {
    // >SEPARATOR customPut
    // Initialize the client
    val client = AnalyticsClient(appId = "YOUR_APP_ID", apiKey = "YOUR_API_KEY", region = Option("YOUR_APP_ID_REGION"))

    // Call the API
    val res = client.customPut[JObject](
      path = "/test/minimal"
    )

    // Use the response
    val value = Await.result(res, Duration(100, "sec"))
    // SEPARATOR<
  }

  /** Snippet for the getAverageClickPosition method.
    *
    * get getAverageClickPosition with minimal parameters
    */
  def snippetForAnalyticsClientGetAverageClickPosition(): Unit = {
    // >SEPARATOR getAverageClickPosition
    // Initialize the client
    val client = AnalyticsClient(appId = "YOUR_APP_ID", apiKey = "YOUR_API_KEY", region = Option("YOUR_APP_ID_REGION"))

    // Call the API
    val res = client.getAverageClickPosition(
      index = "index"
    )

    // Use the response
    val value = Await.result(res, Duration(100, "sec"))
    // SEPARATOR<
  }

  /** Snippet for the getClickPositions method.
    *
    * get getClickPositions with minimal parameters
    */
  def snippetForAnalyticsClientGetClickPositions(): Unit = {
    // >SEPARATOR getClickPositions
    // Initialize the client
    val client = AnalyticsClient(appId = "YOUR_APP_ID", apiKey = "YOUR_API_KEY", region = Option("YOUR_APP_ID_REGION"))

    // Call the API
    val res = client.getClickPositions(
      index = "index"
    )

    // Use the response
    val value = Await.result(res, Duration(100, "sec"))
    // SEPARATOR<
  }

  /** Snippet for the getClickThroughRate method.
    *
    * get getClickThroughRate with minimal parameters
    */
  def snippetForAnalyticsClientGetClickThroughRate(): Unit = {
    // >SEPARATOR getClickThroughRate
    // Initialize the client
    val client = AnalyticsClient(appId = "YOUR_APP_ID", apiKey = "YOUR_API_KEY", region = Option("YOUR_APP_ID_REGION"))

    // Call the API
    val res = client.getClickThroughRate(
      index = "index"
    )

    // Use the response
    val value = Await.result(res, Duration(100, "sec"))
    // SEPARATOR<
  }

  /** Snippet for the getConversationRate method.
    *
    * get getConversationRate with minimal parameters
    */
  def snippetForAnalyticsClientGetConversationRate(): Unit = {
    // >SEPARATOR getConversationRate
    // Initialize the client
    val client = AnalyticsClient(appId = "YOUR_APP_ID", apiKey = "YOUR_API_KEY", region = Option("YOUR_APP_ID_REGION"))

    // Call the API
    val res = client.getConversationRate(
      index = "index"
    )

    // Use the response
    val value = Await.result(res, Duration(100, "sec"))
    // SEPARATOR<
  }

  /** Snippet for the getNoClickRate method.
    *
    * get getNoClickRate with minimal parameters
    */
  def snippetForAnalyticsClientGetNoClickRate(): Unit = {
    // >SEPARATOR getNoClickRate
    // Initialize the client
    val client = AnalyticsClient(appId = "YOUR_APP_ID", apiKey = "YOUR_API_KEY", region = Option("YOUR_APP_ID_REGION"))

    // Call the API
    val res = client.getNoClickRate(
      index = "index"
    )

    // Use the response
    val value = Await.result(res, Duration(100, "sec"))
    // SEPARATOR<
  }

  /** Snippet for the getNoResultsRate method.
    *
    * get getNoResultsRate with minimal parameters
    */
  def snippetForAnalyticsClientGetNoResultsRate(): Unit = {
    // >SEPARATOR getNoResultsRate
    // Initialize the client
    val client = AnalyticsClient(appId = "YOUR_APP_ID", apiKey = "YOUR_API_KEY", region = Option("YOUR_APP_ID_REGION"))

    // Call the API
    val res = client.getNoResultsRate(
      index = "index"
    )

    // Use the response
    val value = Await.result(res, Duration(100, "sec"))
    // SEPARATOR<
  }

  /** Snippet for the getSearchesCount method.
    *
    * get getSearchesCount with minimal parameters
    */
  def snippetForAnalyticsClientGetSearchesCount(): Unit = {
    // >SEPARATOR getSearchesCount
    // Initialize the client
    val client = AnalyticsClient(appId = "YOUR_APP_ID", apiKey = "YOUR_API_KEY", region = Option("YOUR_APP_ID_REGION"))

    // Call the API
    val res = client.getSearchesCount(
      index = "index"
    )

    // Use the response
    val value = Await.result(res, Duration(100, "sec"))
    // SEPARATOR<
  }

  /** Snippet for the getSearchesNoClicks method.
    *
    * get getSearchesNoClicks with minimal parameters
    */
  def snippetForAnalyticsClientGetSearchesNoClicks(): Unit = {
    // >SEPARATOR getSearchesNoClicks
    // Initialize the client
    val client = AnalyticsClient(appId = "YOUR_APP_ID", apiKey = "YOUR_API_KEY", region = Option("YOUR_APP_ID_REGION"))

    // Call the API
    val res = client.getSearchesNoClicks(
      index = "index"
    )

    // Use the response
    val value = Await.result(res, Duration(100, "sec"))
    // SEPARATOR<
  }

  /** Snippet for the getSearchesNoResults method.
    *
    * get getSearchesNoResults with minimal parameters
    */
  def snippetForAnalyticsClientGetSearchesNoResults(): Unit = {
    // >SEPARATOR getSearchesNoResults
    // Initialize the client
    val client = AnalyticsClient(appId = "YOUR_APP_ID", apiKey = "YOUR_API_KEY", region = Option("YOUR_APP_ID_REGION"))

    // Call the API
    val res = client.getSearchesNoResults(
      index = "index"
    )

    // Use the response
    val value = Await.result(res, Duration(100, "sec"))
    // SEPARATOR<
  }

  /** Snippet for the getStatus method.
    *
    * get getStatus with minimal parameters
    */
  def snippetForAnalyticsClientGetStatus(): Unit = {
    // >SEPARATOR getStatus
    // Initialize the client
    val client = AnalyticsClient(appId = "YOUR_APP_ID", apiKey = "YOUR_API_KEY", region = Option("YOUR_APP_ID_REGION"))

    // Call the API
    val res = client.getStatus(
      index = "index"
    )

    // Use the response
    val value = Await.result(res, Duration(100, "sec"))
    // SEPARATOR<
  }

  /** Snippet for the getTopCountries method.
    *
    * get getTopCountries with minimal parameters
    */
  def snippetForAnalyticsClientGetTopCountries(): Unit = {
    // >SEPARATOR getTopCountries
    // Initialize the client
    val client = AnalyticsClient(appId = "YOUR_APP_ID", apiKey = "YOUR_API_KEY", region = Option("YOUR_APP_ID_REGION"))

    // Call the API
    val res = client.getTopCountries(
      index = "index"
    )

    // Use the response
    val value = Await.result(res, Duration(100, "sec"))
    // SEPARATOR<
  }

  /** Snippet for the getTopFilterAttributes method.
    *
    * get getTopFilterAttributes with minimal parameters
    */
  def snippetForAnalyticsClientGetTopFilterAttributes(): Unit = {
    // >SEPARATOR getTopFilterAttributes
    // Initialize the client
    val client = AnalyticsClient(appId = "YOUR_APP_ID", apiKey = "YOUR_API_KEY", region = Option("YOUR_APP_ID_REGION"))

    // Call the API
    val res = client.getTopFilterAttributes(
      index = "index"
    )

    // Use the response
    val value = Await.result(res, Duration(100, "sec"))
    // SEPARATOR<
  }

  /** Snippet for the getTopFilterForAttribute method.
    *
    * get getTopFilterForAttribute with minimal parameters
    */
  def snippetForAnalyticsClientGetTopFilterForAttribute(): Unit = {
    // >SEPARATOR getTopFilterForAttribute
    // Initialize the client
    val client = AnalyticsClient(appId = "YOUR_APP_ID", apiKey = "YOUR_API_KEY", region = Option("YOUR_APP_ID_REGION"))

    // Call the API
    val res = client.getTopFilterForAttribute(
      attribute = "myAttribute",
      index = "index"
    )

    // Use the response
    val value = Await.result(res, Duration(100, "sec"))
    // SEPARATOR<
  }

  /** Snippet for the getTopFiltersNoResults method.
    *
    * get getTopFiltersNoResults with minimal parameters
    */
  def snippetForAnalyticsClientGetTopFiltersNoResults(): Unit = {
    // >SEPARATOR getTopFiltersNoResults
    // Initialize the client
    val client = AnalyticsClient(appId = "YOUR_APP_ID", apiKey = "YOUR_API_KEY", region = Option("YOUR_APP_ID_REGION"))

    // Call the API
    val res = client.getTopFiltersNoResults(
      index = "index"
    )

    // Use the response
    val value = Await.result(res, Duration(100, "sec"))
    // SEPARATOR<
  }

  /** Snippet for the getTopHits method.
    *
    * get getTopHits with minimal parameters
    */
  def snippetForAnalyticsClientGetTopHits(): Unit = {
    // >SEPARATOR getTopHits
    // Initialize the client
    val client = AnalyticsClient(appId = "YOUR_APP_ID", apiKey = "YOUR_API_KEY", region = Option("YOUR_APP_ID_REGION"))

    // Call the API
    val res = client.getTopHits(
      index = "index"
    )

    // Use the response
    val value = Await.result(res, Duration(100, "sec"))
    // SEPARATOR<
  }

  /** Snippet for the getTopSearches method.
    *
    * get getTopSearches with minimal parameters
    */
  def snippetForAnalyticsClientGetTopSearches(): Unit = {
    // >SEPARATOR getTopSearches
    // Initialize the client
    val client = AnalyticsClient(appId = "YOUR_APP_ID", apiKey = "YOUR_API_KEY", region = Option("YOUR_APP_ID_REGION"))

    // Call the API
    val res = client.getTopSearches(
      index = "index"
    )

    // Use the response
    val value = Await.result(res, Duration(100, "sec"))
    // SEPARATOR<
  }

  /** Snippet for the getUsersCount method.
    *
    * get getUsersCount with minimal parameters
    */
  def snippetForAnalyticsClientGetUsersCount(): Unit = {
    // >SEPARATOR getUsersCount
    // Initialize the client
    val client = AnalyticsClient(appId = "YOUR_APP_ID", apiKey = "YOUR_API_KEY", region = Option("YOUR_APP_ID_REGION"))

    // Call the API
    val res = client.getUsersCount(
      index = "index"
    )

    // Use the response
    val value = Await.result(res, Duration(100, "sec"))
    // SEPARATOR<
  }

}

File: snippets/csharp/src/Analytics.cs
using Algolia.Search.Clients;
using Algolia.Search.Http;
using Algolia.Search.Models.Analytics;
using Action = Algolia.Search.Models.Search.Action;

public class SnippetAnalyticsClient
{
  /// <summary>
  /// Snippet for the CustomDelete method.
  ///
  /// allow del method for a custom path with minimal parameters
  /// </summary>
  public async Task SnippetForAnalyticsClientCustomDelete()
  {
    // >SEPARATOR customDelete
    // Initialize the client
    var client = new AnalyticsClient(
      new AnalyticsConfig("YOUR_APP_ID", "YOUR_API_KEY", "YOUR_APP_ID_REGION")
    );

    // Call the API
    var response = await client.CustomDeleteAsync("/test/minimal");
    // SEPARATOR<
  }

  /// <summary>
  /// Snippet for the CustomGet method.
  ///
  /// allow get method for a custom path with minimal parameters
  /// </summary>
  public async Task SnippetForAnalyticsClientCustomGet()
  {
    // >SEPARATOR customGet
    // Initialize the client
    var client = new AnalyticsClient(
      new AnalyticsConfig("YOUR_APP_ID", "YOUR_API_KEY", "YOUR_APP_ID_REGION")
    );

    // Call the API
    var response = await client.CustomGetAsync("/test/minimal");
    // SEPARATOR<
  }

  /// <summary>
  /// Snippet for the CustomPost method.
  ///
  /// allow post method for a custom path with minimal parameters
  /// </summary>
  public async Task SnippetForAnalyticsClientCustomPost()
  {
    // >SEPARATOR customPost
    // Initialize the client
    var client = new AnalyticsClient(
      new AnalyticsConfig("YOUR_APP_ID", "YOUR_API_KEY", "YOUR_APP_ID_REGION")
    );

    // Call the API
    var response = await client.CustomPostAsync("/test/minimal");
    // SEPARATOR<
  }

  /// <summary>
  /// Snippet for the CustomPut method.
  ///
  /// allow put method for a custom path with minimal parameters
  /// </summary>
  public async Task SnippetForAnalyticsClientCustomPut()
  {
    // >SEPARATOR customPut
    // Initialize the client
    var client = new AnalyticsClient(
      new AnalyticsConfig("YOUR_APP_ID", "YOUR_API_KEY", "YOUR_APP_ID_REGION")
    );

    // Call the API
    var response = await client.CustomPutAsync("/test/minimal");
    // SEPARATOR<
  }

  /// <summary>
  /// Snippet for the GetAverageClickPosition method.
  ///
  /// get getAverageClickPosition with minimal parameters
  /// </summary>
  public async Task SnippetForAnalyticsClientGetAverageClickPosition()
  {
    // >SEPARATOR getAverageClickPosition
    // Initialize the client
    var client = new AnalyticsClient(
      new AnalyticsConfig("YOUR_APP_ID", "YOUR_API_KEY", "YOUR_APP_ID_REGION")
    );

    // Call the API
    var response = await client.GetAverageClickPositionAsync("index");
    // SEPARATOR<
  }

  /// <summary>
  /// Snippet for the GetClickPositions method.
  ///
  /// get getClickPositions with minimal parameters
  /// </summary>
  public async Task SnippetForAnalyticsClientGetClickPositions()
  {
    // >SEPARATOR getClickPositions
    // Initialize the client
    var client = new AnalyticsClient(
      new AnalyticsConfig("YOUR_APP_ID", "YOUR_API_KEY", "YOUR_APP_ID_REGION")
    );

    // Call the API
    var response = await client.GetClickPositionsAsync("index");
    // SEPARATOR<
  }

  /// <summary>
  /// Snippet for the GetClickThroughRate method.
  ///
  /// get getClickThroughRate with minimal parameters
  /// </summary>
  public async Task SnippetForAnalyticsClientGetClickThroughRate()
  {
    // >SEPARATOR getClickThroughRate
    // Initialize the client
    var client = new AnalyticsClient(
      new AnalyticsConfig("YOUR_APP_ID", "YOUR_API_KEY", "YOUR_APP_ID_REGION")
    );

    // Call the API
    var response = await client.GetClickThroughRateAsync("index");
    // SEPARATOR<
  }

  /// <summary>
  /// Snippet for the GetConversationRate method.
  ///
  /// get getConversationRate with minimal parameters
  /// </summary>
  public async Task SnippetForAnalyticsClientGetConversationRate()
  {
    // >SEPARATOR getConversationRate
    // Initialize the client
    var client = new AnalyticsClient(
      new AnalyticsConfig("YOUR_APP_ID", "YOUR_API_KEY", "YOUR_APP_ID_REGION")
    );

    // Call the API
    var response = await client.GetConversationRateAsync("index");
    // SEPARATOR<
  }

  /// <summary>
  /// Snippet for the GetNoClickRate method.
  ///
  /// get getNoClickRate with minimal parameters
  /// </summary>
  public async Task SnippetForAnalyticsClientGetNoClickRate()
  {
    // >SEPARATOR getNoClickRate
    // Initialize the client
    var client = new AnalyticsClient(
      new AnalyticsConfig("YOUR_APP_ID", "YOUR_API_KEY", "YOUR_APP_ID_REGION")
    );

    // Call the API
    var response = await client.GetNoClickRateAsync("index");
    // SEPARATOR<
  }

  /// <summary>
  /// Snippet for the GetNoResultsRate method.
  ///
  /// get getNoResultsRate with minimal parameters
  /// </summary>
  public async Task SnippetForAnalyticsClientGetNoResultsRate()
  {
    // >SEPARATOR getNoResultsRate
    // Initialize the client
    var client = new AnalyticsClient(
      new AnalyticsConfig("YOUR_APP_ID", "YOUR_API_KEY", "YOUR_APP_ID_REGION")
    );

    // Call the API
    var response = await client.GetNoResultsRateAsync("index");
    // SEPARATOR<
  }

  /// <summary>
  /// Snippet for the GetSearchesCount method.
  ///
  /// get getSearchesCount with minimal parameters
  /// </summary>
  public async Task SnippetForAnalyticsClientGetSearchesCount()
  {
    // >SEPARATOR getSearchesCount
    // Initialize the client
    var client = new AnalyticsClient(
      new AnalyticsConfig("YOUR_APP_ID", "YOUR_API_KEY", "YOUR_APP_ID_REGION")
    );

    // Call the API
    var response = await client.GetSearchesCountAsync("index");
    // SEPARATOR<
  }

  /// <summary>
  /// Snippet for the GetSearchesNoClicks method.
  ///
  /// get getSearchesNoClicks with minimal parameters
  /// </summary>
  public async Task SnippetForAnalyticsClientGetSearchesNoClicks()
  {
    // >SEPARATOR getSearchesNoClicks
    // Initialize the client
    var client = new AnalyticsClient(
      new AnalyticsConfig("YOUR_APP_ID", "YOUR_API_KEY", "YOUR_APP_ID_REGION")
    );

    // Call the API
    var response = await client.GetSearchesNoClicksAsync("index");
    // SEPARATOR<
  }

  /// <summary>
  /// Snippet for the GetSearchesNoResults method.
  ///
  /// get getSearchesNoResults with minimal parameters
  /// </summary>
  public async Task SnippetForAnalyticsClientGetSearchesNoResults()
  {
    // >SEPARATOR getSearchesNoResults
    // Initialize the client
    var client = new AnalyticsClient(
      new AnalyticsConfig("YOUR_APP_ID", "YOUR_API_KEY", "YOUR_APP_ID_REGION")
    );

    // Call the API
    var response = await client.GetSearchesNoResultsAsync("index");
    // SEPARATOR<
  }

  /// <summary>
  /// Snippet for the GetStatus method.
  ///
  /// get getStatus with minimal parameters
  /// </summary>
  public async Task SnippetForAnalyticsClientGetStatus()
  {
    // >SEPARATOR getStatus
    // Initialize the client
    var client = new AnalyticsClient(
      new AnalyticsConfig("YOUR_APP_ID", "YOUR_API_KEY", "YOUR_APP_ID_REGION")
    );

    // Call the API
    var response = await client.GetStatusAsync("index");
    // SEPARATOR<
  }

  /// <summary>
  /// Snippet for the GetTopCountries method.
  ///
  /// get getTopCountries with minimal parameters
  /// </summary>
  public async Task SnippetForAnalyticsClientGetTopCountries()
  {
    // >SEPARATOR getTopCountries
    // Initialize the client
    var client = new AnalyticsClient(
      new AnalyticsConfig("YOUR_APP_ID", "YOUR_API_KEY", "YOUR_APP_ID_REGION")
    );

    // Call the API
    var response = await client.GetTopCountriesAsync("index");
    // SEPARATOR<
  }

  /// <summary>
  /// Snippet for the GetTopFilterAttributes method.
  ///
  /// get getTopFilterAttributes with minimal parameters
  /// </summary>
  public async Task SnippetForAnalyticsClientGetTopFilterAttributes()
  {
    // >SEPARATOR getTopFilterAttributes
    // Initialize the client
    var client = new AnalyticsClient(
      new AnalyticsConfig("YOUR_APP_ID", "YOUR_API_KEY", "YOUR_APP_ID_REGION")
    );

    // Call the API
    var response = await client.GetTopFilterAttributesAsync("index");
    // SEPARATOR<
  }

  /// <summary>
  /// Snippet for the GetTopFilterForAttribute method.
  ///
  /// get getTopFilterForAttribute with minimal parameters
  /// </summary>
  public async Task SnippetForAnalyticsClientGetTopFilterForAttribute()
  {
    // >SEPARATOR getTopFilterForAttribute
    // Initialize the client
    var client = new AnalyticsClient(
      new AnalyticsConfig("YOUR_APP_ID", "YOUR_API_KEY", "YOUR_APP_ID_REGION")
    );

    // Call the API
    var response = await client.GetTopFilterForAttributeAsync("myAttribute", "index");
    // SEPARATOR<
  }

  /// <summary>
  /// Snippet for the GetTopFiltersNoResults method.
  ///
  /// get getTopFiltersNoResults with minimal parameters
  /// </summary>
  public async Task SnippetForAnalyticsClientGetTopFiltersNoResults()
  {
    // >SEPARATOR getTopFiltersNoResults
    // Initialize the client
    var client = new AnalyticsClient(
      new AnalyticsConfig("YOUR_APP_ID", "YOUR_API_KEY", "YOUR_APP_ID_REGION")
    );

    // Call the API
    var response = await client.GetTopFiltersNoResultsAsync("index");
    // SEPARATOR<
  }

  /// <summary>
  /// Snippet for the GetTopHits method.
  ///
  /// get getTopHits with minimal parameters
  /// </summary>
  public async Task SnippetForAnalyticsClientGetTopHits()
  {
    // >SEPARATOR getTopHits
    // Initialize the client
    var client = new AnalyticsClient(
      new AnalyticsConfig("YOUR_APP_ID", "YOUR_API_KEY", "YOUR_APP_ID_REGION")
    );

    // Call the API
    var response = await client.GetTopHitsAsync("index");
    // SEPARATOR<
  }

  /// <summary>
  /// Snippet for the GetTopSearches method.
  ///
  /// get getTopSearches with minimal parameters
  /// </summary>
  public async Task SnippetForAnalyticsClientGetTopSearches()
  {
    // >SEPARATOR getTopSearches
    // Initialize the client
    var client = new AnalyticsClient(
      new AnalyticsConfig("YOUR_APP_ID", "YOUR_API_KEY", "YOUR_APP_ID_REGION")
    );

    // Call the API
    var response = await client.GetTopSearchesAsync("index");
    // SEPARATOR<
  }

  /// <summary>
  /// Snippet for the GetUsersCount method.
  ///
  /// get getUsersCount with minimal parameters
  /// </summary>
  public async Task SnippetForAnalyticsClientGetUsersCount()
  {
    // >SEPARATOR getUsersCount
    // Initialize the client
    var client = new AnalyticsClient(
      new AnalyticsConfig("YOUR_APP_ID", "YOUR_API_KEY", "YOUR_APP_ID_REGION")
    );

    // Call the API
    var response = await client.GetUsersCountAsync("index");
    // SEPARATOR<
  }
}
